addons/mail/static/src/core/common/thread.js

```diff
@@ -116,7 +116,7 @@ export class Thread extends Component {
                 if (this.props.jumpPresent !== this.lastJumpPresent) {
                     this.messageHighlight?.clearHighlight();
                     if (this.props.thread.loadNewer) {
-                        this.jumpToPresent("instant");
+                        this.jumpToPresent();
```

Comment:
Shouldn't this condition just be removed then? And apply the standard logic of the `else` case (writing scrollTop to either 0 or bottom)?

Reply:
No it's not enough, new test fails with the small delay because the code has to be handled in the scroll adjustment flow to take RPC delay into account like all other scroll adjustment code

Reply:
This branching is still needed for the case where we see an old message and post a new message.
If we simply scroll to bottom, we don't reach presence.

I will prepare a test coverage for this